modules/preferences/main/index.coffee

```diff
@@ -207,7 +207,10 @@ class Preferences extends hx.EventEmitter
       modal: modal
     }
 
-    @locale moment?.locale() or navigator.language or 'en'
+    defaultLocaleId = moment?.locale() or navigator.language or 'en'
```

Comment:
Although it should be always defined, just to be safe it might be best to leave it in as there is no knowing what phantom and slimer will return for navigator.language